cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -236,6 +247,138 @@ func canInstallAndRunAtOnce(mst *initramfsMountsState) (bool, error) {
 	return kernelHasFdeSetup, nil
 }
 
+func getSnap(sysSnaps map[snap.Type]*seed.Snap, snapType snap.Type) (*snap.Info, error) {
+	seedSnap := sysSnaps[snapType]
+	mountPoint := filepath.Join(boot.InitramfsRunMntDir, snapTypeToMountDir[snapType])
+	hooksDir := filepath.Join(mountPoint, "meta", "hooks")
+	return snap.ReadInfoFromMountPoint(seedSnap.SnapName(), mountPoint, seedSnap.Path, hooksDir, seedSnap.SideInfo)
+}
+
+func runFDESetupHook(req *fde.SetupRequest) ([]byte, error) {
+	cmd := exec.Command("fde-setup")
+	stdin, err := cmd.StdinPipe()
+	if err != nil {
+		return make([]byte, 0), err
```

Comment:
I wonder if we could just `return nil, err` here as it would be more idiomatic for go? Or is there a deeper reason? (here and in the other returns in this function).

Reply:
There is no reason other than probably my confusion.